### ISSUE TITLE
Support playback of m3u8 links with additional URL parameters.

### ISSUE THREAD

**Issue by barkoder** [dh0001/hls-dash-extension#2]
Not all m3u8 links end in `.m3u8`

Some m3u8 links are only playable if there are additional URL parameters after it like `.m3u8?foo=bar&variable=value`

Firefox currently just downloads the m3u8 file.

Please support playback of such links through the extension.

Thank you for your work @dh0001

I appreciate it.

Cheers!

**Comment by dh0001** [comment]
Hi :)

Thanks for the report, I've made some changes so URLs with parameters should redirect to the extension, the commit for it is in fb57811

I wasn't able to test with a URL that exclusively doesn't work without parameters, however it does work with https://test-streams.mux.dev/x36xhzz/x36xhzz.m3u8?foo=bar&variable=value#test. Let me know if you have one!

I've uploaded it to the workshop, but i've had problems getting them to accept it before... hopefully it makes it in okay.
